samples/log_callback/README.md

```diff
@@ -0,0 +1,147 @@
+# oe_log_message Callback Sample
+
+This sample demonstrates how to customize log function through the new callback feature of oe_log_message().
+There are several problems with the current logging mechanism that prevent end users from making full use of logging:
+
+- Message can't be redirected to different destinations on demand.
+- Lacking of a method for end users to customize messages.
+- It doesn't have a mechanism to communicate between the host application and oe_log_message() such that end users can't pass more information to the message function.
+
+The callback API is supposed to solve such problems.
+
+## User Experience
+
+By making use of the callback API of oe_log_message(), a user can customize the output of the log message:
+
+- The output destination can be stdout, stderr, or any specified files that the destination is adjustable during runtime.
+- The message can be adjusted according to the log level.
+- The user can get at least as much information about the message as the internal API. For example, log time, enclave or host side, and log level.
+Finally, the user can define a function to customize the log message and can designate the funtion to process all the log messages.
+
+## Changes
```

Comment:
I don't think this section is needed in a samples readme.  Only in the design doc.  Recommend removing.

Reply:
Removed, Thanks!